dev-tools/ls-apis/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "omicron-ls-apis"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+
+[lints]
+workspace = true
+
+[dependencies]
+anyhow.workspace = true
+camino.workspace = true
+cargo_metadata.workspace = true
+clap.workspace = true
+newtype_derive.workspace = true
+parse-display.workspace = true
+petgraph.workspace = true
+serde.workspace = true
+toml.workspace = true
+omicron-workspace-hack.workspace = true
+
+[[bin]]
+name = "ls-apis"
+path = "src/main.rs"
```

Comment:
I don't think you need this bit.

Reply:
Without this, I get:

```
$ cargo xtask ls-apis apis
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.69s
     Running `target/debug/xtask ls-apis apis`
error: no bin target named `ls-apis`.
Available bin targets:
    cert-dev
    ch-dev
    crdb-seed
    db-dev
    dns-db-dump
    dns-server
    dnsadm
    bootstrap
    commtest
    dhcp-server
    gateway-cli
    installinator
    dnswait
    mgs-dev
    clickhouse-admin
    cockroach-admin
    omicron-dev
    mgs
    omicron-ls-apis
    nexus
    schema-updater
    omdb
    omicron-package
    omicron-releng
    sled-agent
    sled-agent-sim
    zone-bundle
    falcon_runner
    falcon_runner_cli
    openapi-manager
    clickhouse-schema-updater
    oximeter
    oxdb
    oximeter-schema
    oxlog
    reconfigurator-cli
    sp-sim
    tufaceous
    wicket
    wicket-dbg
    wicket-dbg-server
    wicketd
    xtask-downloader
    zone-setup
```

This might be an artifact of the directory structure?

Reply:
Fixed in a2d0a6602.